src/solver/optim-model-filler/TimeDependentLinearExpression.cpp

```diff
@@ -143,9 +144,8 @@ size_t TimeDependentLinearExpression::getSize() const
 TimeDependentLinearExpression TimeDependentLinearExpression::shiftLinearExpressions(
   int shiftValue) const
 {
-    const Optimisation::LinearProblemApi::FillContext fillContext{
-      linearExpressions_.begin()->first,
-      linearExpressions_.rbegin()->first};
+    const Optimisation::LinearProblemApi::FillContext
+      fillContext{linearExpressions_.begin()->first, linearExpressions_.rbegin()->first, 0};
```

Comment:
If the 0 isn't used, using `FillContext` seems a bit strange here because we have to provide a 0 artificially. We might as well use the first & last time steps instead of `FillContext`

Reply:
The question is : do we need the year number to time-shift a time-dependent expression ? Answer is prolly no

Reply:
In this case it would be cleaner to refactor in a independent PR IMHO